docs/docs/guides/integrate/identity-providers/azure-ad.md

```diff
@@ -0,0 +1,112 @@
+2. Make sure the permissions include "Microsoft Graph": email, profile and User.Read
+3. The "Other permissions granted" should include "Microsoft Graph: openid"
+
+![Azure API permissions](/img/guides/azure_api_permissions.png)
+
+## ZITADEL Configuration
+
+### Create new Azure AD Provider
+
+Go to the settings of your ZITADEL instance or the organization where you like to add a new **Azure AD** provider.
+Choose the **Microsoft** provider template. This template has everything you need preconfigured. You only have to add the client ID and secret, you have created in the step before.
+
+You can configure the following settings if you like, a useful default will be filled if you don't change anything:
+
+**Scopes**: The scopes define which scopes will be sent to the provider, `openid`, `profile`, and `email` are prefilled. This information will be taken to create/update the user within ZITADEL. Make sure to alswo add `User.Read`
+
+**Email Verified**: Azure AD doesn't send the email verified claim in the token of the user, if you don't enable this setting, the user will be created with the email not verified, which results in a Email Verification Message for the user. If you want to avoid that, make sure to enable "Email verified". In that case the user will be created with a verified email address.
+
+**Tenant Type**: The tenant type should be configured according what you have chosen in the settings of your Azure Ad application previously.
+- Common: Choose common if you want all microsoft accounts being able to login. In this case you have configured "Accounts in any organizational directory and personal Microsoft accounts" in your Azure App configuration
+- Organizations: Choose organization if you have Azure AD Tenants and no personal accounts. (You have configured either "Accounts in this organization" or "Accounts in any organizational directory" on your Azure APP)
+- Consumers: Choose this if you want to allow public accounts. (In your Azure AD App you have configured "Personal Microsoft accounts only")
+
+**Tenant ID**: If you have selected either "organizations" or "Customers" in the Tenant Type, you have to enter the Directory (Tenant) ID, copied previously in the Azure App configuration, here.
+
+**Automatic creation**: If this setting is enabled the user will be created automatically within ZITADEL, if it doesn't exist.
+
+**Automatic update**: If this setting is enabled, the user will be updated within ZITADEL, if some user data are changed withing the provider. E.g if the lastname changes on the GitHub account, the information will be changed on the ZITADEL account on the next login. 
+
+**Account creation allowed**: This setting determines if account creation within ZITADEL is allowed or not.
+
+**Account linking allowed**: This setting determines if account linking is allowed. (E.g an account within ZITADEL should already be existing and the when login with GitHub an account should be linked)
```

Comment:
Can we put this in its own mdx file and reuse it in every IdP guide?

Reply:
Makes sense, I changed it